homeassistant/bootstrap.py

```diff
@@ -422,4 +432,11 @@ async def async_setup_multi_components(domains: Set[str]) -> None:
         await async_setup_multi_components(stage_2_domains)
 
     # Wrap up startup
-    await hass.async_block_till_done()
+    try:
+        async with timeout(TIMEOUT_EVENT_BOOTSTRAP):
```

Comment:
This seems to do a very similar thing as #36082. I think that we should keep it out of this PR as it's already impactful enough.

Reply:
I'm concerned about taking it out as the user can now interact with the api/ui and trigger something in an integration that blocks for a long time like websocket did.  Taking it out might rehash #6973